### PR TITLE
Updated dl, dt, dd elements

### PR DESCRIPTION
**Pull request by mfuji09**
- updated the explanation when dl includes div
 (W3C HTML also defines it since HTML 5.2, and the div should be children of dl)
- fixed "Examples", "See also", etc. for consistency
- Adjust tags

### CODE REVIEW

File: files/en-us/web/html/element/dl/index.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 title: '<dl>: The Description List element'
 slug: Web/HTML/Element/dl
 tags:
+  - Difinition List
```
Comment (chrisdavidmills) [inline files/en-us/web/html/element/dl/index.html:5]: Typo - should be "Definition list"

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
One small change request ;-)

File: files/en-us/web/html/element/dl/index.html (hunk 1 above)
Comment (mfuji09) [reply files/en-us/web/html/element/dl/index.html]: Oh! It's a typo. I have corrected.
Thank you!

**Review by chrisdavidmills — APPROVED** [review]
Perfect, thanks!

**Pull request merged**